src/components/City.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import LocationWeather from "./LocationWeather";
+
+const City = ({ cityInfo }) => {
+  const { countryName, cityName, mapUrl, state, latLng } = cityInfo;
+  return (
+    <div className="box mb-1">
+      <div className="container  text-center  ">
+        {/* <Link to={`${url}/${cityName}`}> */}
+        <img src={mapUrl} alt={`map of ${cityName} `} width="150" />
+        {/* </Link> */}
```

Comment:
- [ ] Please remove any commented-out code to keep your code clean and optimized. 😄 🙏 
>Please apply this request to all similar situations through your code files. 🙏

Reply:
Well Done with fixes !!!